api/overview/azure/service-bus.md

```diff
@@ -43,10 +61,31 @@ Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.Manag
 Install-Package Microsoft.Azure.Management.ServiceBus.Fluent
 ```
 
+#### .NET Core CLI
+
 ```bash
 dotnet add package Microsoft.Azure.Management.ServiceBus.Fluent
 ```
 
+### Code Example
+
+This example creates a Service Bus queue with a maximum size of 1024MB.
+
+```csharp
+// using Microsoft.Azure.Management.ServiceBus.Fluent;
+// using Microsoft.Azure.Management.ServiceBus.Fluent.Models;
+
+using (ServiceBusManagementClient client = new ServiceBusManagementClient(credentials))
+{
+    client.SubscriptionId = subscriptionId;
+    QueueInner parameters = new QueueInner
+    {
+        MaxSizeInMegabytes = 1024
+    };
+    await client.Queues.CreateOrUpdateAsync(resourceGroupName, namespaceName, queueName, parameters);
+}
+```
+
 > [!div class="nextstepaction"]
 > [Explore the management APIs](/dotnet/api/overview/azure/servicebus/management)
 

```

Comment:
I think you need to add these to the end of the overview

[PackageManager]: https://docs.microsoft.com/nuget/tools/package-manager-console  
[DotNetCLI]: https://docs.microsoft.com/dotnet/core/tools/dotnet-add-package